client/state/data-layer/wpcom/me/settings/index.js

```diff
@@ -22,16 +22,15 @@ import { USER_SETTINGS_REQUEST, USER_SETTINGS_SAVE } from 'state/action-types';
  * Decodes entities in those specific user settings properties
  * that the REST API returns already HTML-encoded
  */
+const PROPERTIES_TO_DECODE = [ 'display_name', 'description', 'user_URL' ];
```

Comment:
maybe just use a Set?

Reply:
With the goal of making the property name lookup faster?

Reply:
I wonder what the perf difference would be for a list of size 3 like this. Is the O(N) operation (`includes`) faster for very small N?  

I'd check jsperf but I think it is down right now for some reason :/

Reply:
A linear search in a list of 3 strings should be always faster than a hash table lookup. I think the fastest implementation would be a `!! PROPS[ name ]` lookup in `PROPS = { 'display_name': true, ... }`. For small objects like this, the property lookup is a linear search.

Reply:
no, not speed, just semantics. and you don't have to pull in `includes` to do it.

Reply:
OK, let's use `Set` then!

Reply:
Agreed that perf is a non-concern :smile:

`Set` is nice though and actually still could be faster technically than a list lookup just because it's able to jump faster into C++ due to the more-constrained semantics - wouldn't matter though.